torch/_inductor/fx_passes/group_batch_fusion.py

```diff
@@ -740,46 +752,105 @@ def __init__(self, **kwargs):
         super().__init__(aten.mul.Tensor, **kwargs)
 
 
+class _OrderedSet:
+    def __init__(self, param=None):
+        if param:
+            self.rep = OrderedDict({k: None for k in param})
```

Comment:
we should kill OrderedDict from the codebase now that dicts are ordered

Reply:
Or we could keep it to indicate that the ordering is part of the expected behavior and not just a side-effect...